### DISCUSSION TITLE
Adding a track to an existing album. How?

### DISCUSSION THREAD

**Discussion by neilbaldwin** [beetbox/beets#4665] [Q&A] [answered]
I'd accidentally deleted a file from an album folder *before* importing it with Beets. At the time I just added the tracks anyway and so the album exists in my Beets database with a track missing.

I now have the missing track (restored from an old backup drive) and I want to add it to the album. I can't seem to figure out how to do that though.

`beet import` on the file tries to import it as an album and, of course, Beets thinks that all the other tracks are missing.

Is there a clever way to do it? Only way I can see at this point it to move the files, remove the album from Beets and re-import it?

Thanks.

**Comment by jackwilsdon** [comment, marked as answer]
You can just re-import the album as you did but tell beets to merge it with your existing one when prompted - it shouldn't matter that there are tracks missing. [This section](https://beets.readthedocs.io/en/stable/guides/tagger.html#duplicates) of the auto-tagger documentation seems relevant.

**Comment by neilbaldwin** [comment]
Thanks for you answer. I did try that before posting as I've seen that situation before (the choice to merge). In this instance it didn't seem to work. I ended up removing and re-importing this time but would be interested if there is a proper way to do this (or maybe it's how you describe by just re-importing).

**Reply by JOJ0** [reply]
It works exactly as @jackwilsdon explainend. Reimport that one file and choose exactly the match you chose for your existing album. Dont let the importer fool you that all tracks except one are missing. Choose merge when asked ooor just set duplicate_action: merge in your config to do that automatically. I've set my conf that way since it often happens that I find single tracks I'd like to including into an album.

**Comment by neilbaldwin** [comment]
I tried again but this time had some faith in the process after the prompt that tells you that the album has missing tracks and you then get the opportunity to merger the new album with the old. Works perfectly. I just needed to believe in Beets more!
